mysql-test/main/server_maturity_in_version.test

```diff
@@ -0,0 +1,5 @@
+--echo #
+--echo # MDEV - 14558: Indicate maturity level of the server.
+--echo #
+SELECT @@version;
```

Comment:
So every version bump doesn't need to change this test, can you do `SELECT @@version RLIKE ....` (or another regex) to make the expression just return 1 if successful. Same with `SELECT version()` below.

Reply:
the regex also needs to account this may not be a debug build.